lib/still/compiler/compilation_stage.ex

```diff
@@ -96,13 +96,23 @@ defmodule Still.Compiler.CompilationStage do
      }}
   end
 
-  @impl true
-  def handle_info(:run, %{to_compile: [], changed: true} = state) do
+  def handle_info(:notify_subscribers, %{to_compile: []} = state) do
     state.subscribers
     |> Enum.each(fn pid ->
       send(pid, :bus_empty)
     end)
 
+    {:noreply, state}
+  end
+
+  def handle_info(:notify_subscribers, state) do
+    {:noreply, state}
+  end
+
+  @impl true
```

Comment:
`@impl true` should be in the first branch, right?

Reply:
👍